server/src/internalClusterTest/java/org/opensearch/search/functionscore/QueryRescorerIT.java

```diff
@@ -128,7 +128,7 @@ public void testEnforceWindowSize() throws InterruptedException {
                     new QueryRescorerBuilder(
                         functionScoreQuery(matchAllQuery(), ScoreFunctionBuilders.weightFactorFunction(100)).boostMode(
                             CombineFunction.REPLACE
-                        )
+                        ).queryName("hello world")
```

Comment:
We added query name but have not added any assertions?

Reply:
added an assertion